plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go

```diff
@@ -171,9 +172,9 @@ func TestWinPerfCountersConfigGet2Integration(t *testing.T) {
 	require.True(t, ok)
 
 	if len(hostCounters.counters) == 0 {
-		require.FailNow(t, "no results returned from the counterPath: %v", len(hostCounters.counters))
+		require.FailNow(t, fmt.Sprintf("No results returned from the counterPath: %v", len(hostCounters.counters)))
 	} else if len(hostCounters.counters) > 1 {
-		require.FailNow(t, "too many results returned from the counterPath: %v", len(hostCounters.counters))
+		require.FailNow(t, fmt.Sprintf("Too many results returned from the counterPath: %v", len(hostCounters.counters)))
```

Comment:
`require.FailNowf`! Anyway, this will prevent any deferred function to run IIRC. Shouldn't we simply test with
`require.Len(t, hostCounters.counters, 1)`? Same for the other tests below.